src/Microsoft.DocAsCode.Build.Engine/PostProcessors/PostProcessorsHandlerWithIncremental.cs

```diff
@@ -42,8 +46,152 @@ public void Handle(List<PostProcessor> postProcessors, Manifest manifest, string
                 throw new ArgumentNullException(nameof(outputFolder));
             }
 
-            // TODO: incremental things
-            _innerHandler.Handle(postProcessors, manifest, outputFolder);
+            // TODO: add logs
+            // TODO: replay warning
+            var increItems = manifest.Files.Where(i => i.IsIncremental).ToList();
+            var noneIncreItems = manifest.Files.Where(i => !i.IsIncremental).ToList();
+
+            PreHandle(manifest, outputFolder, increItems, noneIncreItems);
+            {
```

Comment:
To group inner as core logic...is optional here

---
In reply to: [95303118](https://github.com/dotnet/docfx/pull/1136#discussion_r95303118) [](ancestors = 95303118)